itl/plugins-contrib.d/smart-attributes.conf

```diff
@@ -8,15 +8,46 @@ object CheckCommand "smart-attributes" {
 	command = [ PluginContribDir + "/check_smart_attributes" ]
 
 	arguments = {
-		"-dbj" = {
+		"--path" = {
+			value = "$smart_attributes_smartclt_path$"
+			description = "Specify the path at which the smartctl binary can be found."
+		}
+		"--device" = {
+			value = "$smart_attributes_device$"
+			description = "Insert the device name (e.g. /dev/sda) to monitor"
+		}
+		"--regex" = {
+			value = "$smart_attributes_device_regex$"
+			description = "Insert a regex to finde devices to monitor"
+		}
+		"--dbjson" = {
 			required = true
 			value = "$smart_attributes_config_path$"
 			description = "Path to the smart attributes config file (e.g. check_smartdb.json)"
 		}
-		"-d" = {
-			required = true
-			value = "$smart_attributes_device$"
-			description = "Insert the device name (e.g. /dev/sda) to monitor"
+		"--ucfgjson" = {
+			value = "$smart_attributes_user_config_path$"
+			description = "Specify the path at which the JSON smart db can be found."
+
+		}
+		"--critical_on_unknown" = {
+			set_if = "$smart_attributes_critical_if_not_found$"
+			description = "Show critical if the device was not found in database."
+		}
+		"-nosudo" = {
```

Comment:
Please consider adding a comment after this line indicating that the single dash is valid for `-nosudo` and not a typo.

Reply:
Thanks for adding the comment.